packages/tools/devtools/devtools-view/api-report/devtools-view.alpha.api.md

```diff
@@ -4,8 +4,6 @@
 
 ```ts
 
-export { IMessageRelay }
```

Comment:
This type has been `@internal` since its inception in 2023 and actually lives in `@fluidframework/devtools-core`, this package just re-exports it. Unclear why it was showing up here when it's internal, but it seems correct that it is not anymore.

FYI @Josmithr & @jason-ha, just informational since you both have a decent amount of knowledge/involvement with api-extractor.

Reply:
This is expected. I fixed this bug a week or two back, we just hadn't ingested the change yet. Glad it's working!
https://github.com/microsoft/rushstack/pull/5211